here-naksha-lib-psql/src/commonMain/kotlin/naksha/psql/executors/write/UpdateFeature.kt

```diff
@@ -123,10 +125,11 @@ class UpdateFeature(
         }
     }
 
-    private fun insertPreviousVersionToHst(
+    protected fun insertHeadVersionToHst(
```

Comment:
The name is actually "insert HEAD version to Hst" so it is correct. But I will find better expressions to avoid this misunderstanding about version

Reply:
As said, I'm okay with the method name (even while I prefer mine ;-)), but the argument name is very misleading, please rename it back from `versionInHst` to `nextVersion`, because the version is not changed. However, this is only a minor issue, after all.